pkgs/applications/audio/open-stage-control/default.nix

```diff
@@ -11,10 +11,10 @@ buildNpmPackage rec {
     hash = "sha256-XgwlRdwUSl4gIRKqk6BnMAKarVvp291zk8vmNkuRWKo=";
   };
 
-  patches = [
-    # Use generated package-lock.json since upstream does not provide one in releases
-    ./package-lock.json.patch
-  ];
+  # Use generated package-lock.json since upstream does not provide one in releases
+  postPatch = ''
+    cp -T ${./package-lock.json} ./package-lock.json
```

Comment:
Is now ``ln -s`` working?

Reply:
It should be!

I didn't want there to be a chance that the package-lock.json link ends up in some output and ends up having nixpkgs in the closure though

I'll do some testing and see if that actually happens or not. If not, I don't mind changing it

Reply:
Oh yeah Nix copies the individual file for these types of references anyway so it doesn't matter

Aside from that, the testing showed that `package-lock.json` isn't packed in `npm pack` and so it isn't present in the output anyway with the default install hook or the custom phase here

So yeah I'll push the change in a moment!